[2024-10-07T22:08:05.601-0300] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-10-07T22:08:05.637-0300] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: preprocess_predict.preprocess manual__2024-10-08T01:08:04.858539+00:00 [queued]>
[2024-10-07T22:08:05.648-0300] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: preprocess_predict.preprocess manual__2024-10-08T01:08:04.858539+00:00 [queued]>
[2024-10-07T22:08:05.649-0300] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2024-10-07T22:08:05.674-0300] {taskinstance.py:2330} INFO - Executing <Task(_PythonVirtualenvDecoratedOperator): preprocess> on 2024-10-08 01:08:04.858539+00:00
[2024-10-07T22:08:05.683-0300] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=1981) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-10-07T22:08:05.684-0300] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'preprocess_predict', 'preprocess', 'manual__2024-10-08T01:08:04.858539+00:00', '--job-id', '55', '--raw', '--subdir', 'DAGS_FOLDER/Predict/preprocess_predict.py', '--cfg-path', '/tmp/tmpe1opo8vv']
[2024-10-07T22:08:05.685-0300] {standard_task_runner.py:63} INFO - Started process 1982 to run task
[2024-10-07T22:08:05.686-0300] {standard_task_runner.py:91} INFO - Job 55: Subtask preprocess
[2024-10-07T22:08:05.752-0300] {task_command.py:426} INFO - Running <TaskInstance: preprocess_predict.preprocess manual__2024-10-08T01:08:04.858539+00:00 [running]> on host 4a07270c49a4
[2024-10-07T22:08:05.914-0300] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='preprocess_predict' AIRFLOW_CTX_TASK_ID='preprocess' AIRFLOW_CTX_EXECUTION_DATE='2024-10-08T01:08:04.858539+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-10-08T01:08:04.858539+00:00'
[2024-10-07T22:08:05.916-0300] {taskinstance.py:430} INFO - ::endgroup::
[2024-10-07T22:08:05.918-0300] {process_utils.py:183} INFO - Executing cmd: /home/***/.local/bin/python -m virtualenv /tmp/venvxju9iltu --python=python
[2024-10-07T22:08:05.919-0300] {process_utils.py:187} INFO - Output:
[2024-10-07T22:08:06.681-0300] {process_utils.py:191} INFO - created virtual environment CPython3.12.3.final.0-64 in 204ms
[2024-10-07T22:08:06.682-0300] {process_utils.py:191} INFO -   creator CPython3Posix(dest=/tmp/venvxju9iltu, clear=False, no_vcs_ignore=False, global=False)
[2024-10-07T22:08:06.683-0300] {process_utils.py:191} INFO -   seeder FromAppData(download=False, pip=bundle, via=copy, app_data_dir=/home/***/.local/share/virtualenv)
[2024-10-07T22:08:06.684-0300] {process_utils.py:191} INFO -     added seed packages: pip==24.2
[2024-10-07T22:08:06.685-0300] {process_utils.py:191} INFO -   activators BashActivator,CShellActivator,FishActivator,NushellActivator,PowerShellActivator,PythonActivator
[2024-10-07T22:08:06.731-0300] {process_utils.py:183} INFO - Executing cmd: /tmp/venvxju9iltu/bin/pip install -r /tmp/venvxju9iltu/requirements.txt
[2024-10-07T22:08:06.733-0300] {process_utils.py:187} INFO - Output:
[2024-10-07T22:08:08.297-0300] {process_utils.py:191} INFO - Collecting joblib (from -r /tmp/venvxju9iltu/requirements.txt (line 1))
[2024-10-07T22:08:08.300-0300] {process_utils.py:191} INFO -   Using cached joblib-1.4.2-py3-none-any.whl.metadata (5.4 kB)
[2024-10-07T22:08:08.660-0300] {process_utils.py:191} INFO - Collecting numpy (from -r /tmp/venvxju9iltu/requirements.txt (line 2))
[2024-10-07T22:08:08.663-0300] {process_utils.py:191} INFO -   Using cached numpy-2.1.2-cp312-cp312-manylinux_2_17_x86_64.manylinux2014_x86_64.whl.metadata (60 kB)
[2024-10-07T22:08:08.910-0300] {process_utils.py:191} INFO - Collecting pandas (from -r /tmp/venvxju9iltu/requirements.txt (line 3))
[2024-10-07T22:08:08.914-0300] {process_utils.py:191} INFO -   Using cached pandas-2.2.3-cp312-cp312-manylinux_2_17_x86_64.manylinux2014_x86_64.whl.metadata (89 kB)
[2024-10-07T22:08:09.084-0300] {process_utils.py:191} INFO - Collecting scikit-learn (from -r /tmp/venvxju9iltu/requirements.txt (line 4))
[2024-10-07T22:08:09.086-0300] {process_utils.py:191} INFO -   Using cached scikit_learn-1.5.2-cp312-cp312-manylinux_2_17_x86_64.manylinux2014_x86_64.whl.metadata (13 kB)
[2024-10-07T22:08:09.133-0300] {process_utils.py:191} INFO - Collecting python-dateutil>=2.8.2 (from pandas->-r /tmp/venvxju9iltu/requirements.txt (line 3))
[2024-10-07T22:08:09.135-0300] {process_utils.py:191} INFO -   Using cached python_dateutil-2.9.0.post0-py2.py3-none-any.whl.metadata (8.4 kB)
[2024-10-07T22:08:09.207-0300] {process_utils.py:191} INFO - Collecting pytz>=2020.1 (from pandas->-r /tmp/venvxju9iltu/requirements.txt (line 3))
[2024-10-07T22:08:09.209-0300] {process_utils.py:191} INFO -   Using cached pytz-2024.2-py2.py3-none-any.whl.metadata (22 kB)
[2024-10-07T22:08:09.242-0300] {process_utils.py:191} INFO - Collecting tzdata>=2022.7 (from pandas->-r /tmp/venvxju9iltu/requirements.txt (line 3))
[2024-10-07T22:08:09.244-0300] {process_utils.py:191} INFO -   Using cached tzdata-2024.2-py2.py3-none-any.whl.metadata (1.4 kB)
[2024-10-07T22:08:09.417-0300] {process_utils.py:191} INFO - Collecting scipy>=1.6.0 (from scikit-learn->-r /tmp/venvxju9iltu/requirements.txt (line 4))
[2024-10-07T22:08:09.418-0300] {process_utils.py:191} INFO -   Using cached scipy-1.14.1-cp312-cp312-manylinux_2_17_x86_64.manylinux2014_x86_64.whl.metadata (60 kB)
[2024-10-07T22:08:09.459-0300] {process_utils.py:191} INFO - Collecting threadpoolctl>=3.1.0 (from scikit-learn->-r /tmp/venvxju9iltu/requirements.txt (line 4))
[2024-10-07T22:08:09.461-0300] {process_utils.py:191} INFO -   Using cached threadpoolctl-3.5.0-py3-none-any.whl.metadata (13 kB)
[2024-10-07T22:08:09.488-0300] {process_utils.py:191} INFO - Collecting six>=1.5 (from python-dateutil>=2.8.2->pandas->-r /tmp/venvxju9iltu/requirements.txt (line 3))
[2024-10-07T22:08:09.490-0300] {process_utils.py:191} INFO -   Using cached six-1.16.0-py2.py3-none-any.whl.metadata (1.8 kB)
[2024-10-07T22:08:09.510-0300] {process_utils.py:191} INFO - Using cached joblib-1.4.2-py3-none-any.whl (301 kB)
[2024-10-07T22:08:09.512-0300] {process_utils.py:191} INFO - Using cached numpy-2.1.2-cp312-cp312-manylinux_2_17_x86_64.manylinux2014_x86_64.whl (16.0 MB)
[2024-10-07T22:08:09.529-0300] {process_utils.py:191} INFO - Using cached pandas-2.2.3-cp312-cp312-manylinux_2_17_x86_64.manylinux2014_x86_64.whl (12.7 MB)
[2024-10-07T22:08:09.548-0300] {process_utils.py:191} INFO - Using cached scikit_learn-1.5.2-cp312-cp312-manylinux_2_17_x86_64.manylinux2014_x86_64.whl (12.9 MB)
[2024-10-07T22:08:09.569-0300] {process_utils.py:191} INFO - Using cached python_dateutil-2.9.0.post0-py2.py3-none-any.whl (229 kB)
[2024-10-07T22:08:09.571-0300] {process_utils.py:191} INFO - Using cached pytz-2024.2-py2.py3-none-any.whl (508 kB)
[2024-10-07T22:08:09.574-0300] {process_utils.py:191} INFO - Using cached scipy-1.14.1-cp312-cp312-manylinux_2_17_x86_64.manylinux2014_x86_64.whl (40.8 MB)
[2024-10-07T22:08:09.617-0300] {process_utils.py:191} INFO - Using cached threadpoolctl-3.5.0-py3-none-any.whl (18 kB)
[2024-10-07T22:08:09.619-0300] {process_utils.py:191} INFO - Using cached tzdata-2024.2-py2.py3-none-any.whl (346 kB)
[2024-10-07T22:08:09.621-0300] {process_utils.py:191} INFO - Using cached six-1.16.0-py2.py3-none-any.whl (11 kB)
[2024-10-07T22:08:09.816-0300] {process_utils.py:191} INFO - Installing collected packages: pytz, tzdata, threadpoolctl, six, numpy, joblib, scipy, python-dateutil, scikit-learn, pandas
[2024-10-07T22:08:23.430-0300] {process_utils.py:191} INFO - Successfully installed joblib-1.4.2 numpy-2.1.2 pandas-2.2.3 python-dateutil-2.9.0.post0 pytz-2024.2 scikit-learn-1.5.2 scipy-1.14.1 six-1.16.0 threadpoolctl-3.5.0 tzdata-2024.2
[2024-10-07T22:08:23.850-0300] {process_utils.py:183} INFO - Executing cmd: /tmp/venvxju9iltu/bin/python /tmp/venv-callm55xu87w/script.py /tmp/venv-callm55xu87w/script.in /tmp/venv-callm55xu87w/script.out /tmp/venv-callm55xu87w/string_args.txt /tmp/venv-callm55xu87w/termination.log
[2024-10-07T22:08:23.852-0300] {process_utils.py:187} INFO - Output:
[2024-10-07T22:08:25.122-0300] {process_utils.py:191} INFO - Traceback (most recent call last):
[2024-10-07T22:08:25.123-0300] {process_utils.py:191} INFO -   File "/tmp/venv-callm55xu87w/script.py", line 66, in <module>
[2024-10-07T22:08:25.124-0300] {process_utils.py:191} INFO -     res = task_preprocess(*arg_dict["args"], **arg_dict["kwargs"])
[2024-10-07T22:08:25.126-0300] {process_utils.py:191} INFO -           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
[2024-10-07T22:08:25.127-0300] {process_utils.py:191} INFO -   File "/tmp/venv-callm55xu87w/script.py", line 21, in task_preprocess
[2024-10-07T22:08:25.128-0300] {process_utils.py:191} INFO -     from ***.operators.python import get_current_context
[2024-10-07T22:08:25.129-0300] {process_utils.py:191} INFO - ModuleNotFoundError: No module named '***'
[2024-10-07T22:08:25.545-0300] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-10-07T22:08:25.555-0300] {taskinstance.py:2905} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 465, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 432, in _execute_callable
    return execute_callable(context=context, **execute_callable_kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 400, in wrapper
    return func(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/decorators/base.py", line 265, in execute
    return_value = super().execute(context)
                   ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 400, in wrapper
    return func(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/operators/python.py", line 437, in execute
    return super().execute(context=serializable_context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 400, in wrapper
    return func(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/operators/python.py", line 235, in execute
    return_value = self.execute_callable()
                   ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/operators/python.py", line 759, in execute_callable
    result = self._execute_python_callable_in_subprocess(python_path)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/operators/python.py", line 514, in _execute_python_callable_in_subprocess
    raise AirflowException(error_msg) from None
airflow.exceptions.AirflowException: Process returned non-zero exit status 1.
No module named '***'
[2024-10-07T22:08:25.560-0300] {taskinstance.py:1206} INFO - Marking task as UP_FOR_RETRY. dag_id=preprocess_predict, task_id=preprocess, run_id=manual__2024-10-08T01:08:04.858539+00:00, execution_date=20241008T010804, start_date=20241008T010805, end_date=20241008T010825
[2024-10-07T22:08:25.586-0300] {standard_task_runner.py:110} ERROR - Failed to execute job 55 for task preprocess (Process returned non-zero exit status 1.
No module named '***'; 1982)
[2024-10-07T22:08:25.598-0300] {local_task_job_runner.py:240} INFO - Task exited with return code 1
[2024-10-07T22:08:25.620-0300] {taskinstance.py:3498} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-10-07T22:08:25.623-0300] {local_task_job_runner.py:222} INFO - ::endgroup::
